lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js

```diff
@@ -95,7 +95,7 @@ Then('the user is shown the error message {string}', (errorMessage) => {
 });
 
 Then(`the user remains in the Accuracy of the appellant's submission page`, () => {
-  cy.verifyPageTitle(pageTitle);
+  cy.title().should('include', 'Error:');
```

Comment:
A small potential improvement here might be to use a regex match:

`cy.title().should('match', /^Error: /);`

The benefit to this is that it checks the string starts with 'Error:', whereas the `include` approach would match:

* 'There is an Error: here'
* 'Error: something'
* 'Error:'

And so on. Depends if you need to be that specific.

Reply:
Updated as per review comments